lib/modules/userTagger.js

```diff
@@ -416,7 +473,30 @@ function populateDialog(tag: Tag, card) {
 		elements.preview.appendChild(Tag.buildTagElement(extract()));
 	}
 
+	function buildPresetTagElement(text: string, color: string) {
+		const element = Tag.buildTagElement({ text, color });
+		element.addEventListener('click', () => {
+			tag.load({ text, color });
+			tag.save();
+			card.close();
+		});
+		return element;
+	}
+
+	function saveAsPreset() {
+		const { text, color } = extract();
+		if (text && color) {
```

Comment:
Alert user if this conditional fails

Reply:
How should the alert be set up?

Reply:
A simple `window.alert` will probably suffice.

Reply:
Actually right now it seems like you're able to set up a preset tag in the settings console with no text. That will show up in the dialog under presets but tagged users won't show up on the dashboard since the text is considered null. 

So should users be able to set up tags without text, or if not, is there a way to validate that the fields in the preset table in the userTagger settings console are not empty?

Additionally it seems somewhere in my changes I have introduced a new bug that allows you to save a tag without any text. I don't really see where my additions to userTagger would cause this to happen but I'm taking a closer look. Also I'm not getting that issue in v5.8.6 on Chrome though I haven't checked out v5.9.0 or master yet.

Reply:
Update: Looks like that bug is in `master` too but I've only tested it out with Firefox.

Reply:
FWIW it is intended that you can create tags with no text, in case you just want to use colors to denote different users.